### PR TITLE
More efficient parsing of Mr. Sandman output file

### PR DESCRIPTION
**Pull request by nathan-weinberg**
Based off convos with @paigerube14 

Also did some minor cleanup across our stack

### CODE REVIEW

**Comment by nathan-weinberg** [comment]
Tested here: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/nweinber-e2e-benchmarking-multibranch-pipeline/job/netobserv-perf-tests/346/

File: scripts/deploy-loki-aws-secret.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-LOKI_BUCKET_NAME=${1:-netobserv-loki-ocpqe-perf}
+LOKI_BUCKET_NAME=${1:-netobserv-ocpqe-default}
```
Comment (paigerube14) [inline scripts/deploy-loki-aws-secret.sh:3]: do you want these changes as part of this pR?

Comment (nathan-weinberg) [reply scripts/deploy-loki-aws-secret.sh:3]: Yes - generally this bit is wrapped by code within `netobserv.sh` - this is just a legacy piece of code I wanted to update while I was here

**Comment by memodi** [comment]
@nathan-weinberg - Sorry, I am not following this PR, which part is doing efficient parsing of output file?

**Comment by nathan-weinberg** [comment]
> @nathan-weinberg - Sorry, I am not following this PR, which part is doing efficient parsing of output file?

Instead of making 5 separate `jq` calls to parse data from `workload.json` we are doing it in a single line now

**Pull request merged**